### DISCUSSION TITLE
Is it okay serve sitemap.xml in public folder

### DISCUSSION THREAD

**Discussion by bexoss** [vercel/next.js#15009] [Help] [answered]
Can I put my `sitemap.xml` to inside of public folder?

I can see sitemap when I access `http://mywebsite.com/sitemap.xml`

I wonder should I add some options like `'Content-Type', 'application/xml'`.

If so, How could I do it?

**Comment by srikanth912** [comment, marked as answer]
It is perfectly fine to put sitemap.xml inside public and it will be automatically served with `Content-Type: application/xml `
